addons/website_sale/controllers/cart.py

```diff
@@ -58,7 +58,7 @@ def cart(self, id=None, access_token=None, revive_method='', **post):
             values['suggested_products'] = order_sudo._cart_accessories()
             values.update(self._get_express_shop_payment_values(order_sudo))
 
-        values.update(request.website._get_checkout_step_values('/shop/cart'))
+        values.update(request.website._get_checkout_step_values(request.httprequest.path))
```

Comment:
Since the call to the line is not the same every time, maybe we don't need to give `request.httprequest.path` as an args and it could be set as a default in the method? (I don't know if it would work with request)

Reply:
I set href=None as default parameter and set it only when different than request.httprequest.path.